tutorials/polkadot-sdk/system-chains/asset-hub/send-a-tx-paying-fee-different-token.md

```diff
@@ -0,0 +1,167 @@
+
+## Implementation
+
+In the following sub-sections, you’ll set the stage with imports and a few constants, introduce the transaction signer, and connect to the Polkadot Asset Hub chain. Then you’ll craft a simple DOT transfer and send it, asking the network to charge the fee in USDT instead of DOT.
+
+### Import Dependencies
+
+Add the following imports to your `fee-payment-transaction.ts` file:
+
+```typescript title="fee-payment-transaction.ts"
+--8<-- "code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction.ts:1:12"
+```
+
+### Define Constants
+
+Define the constants for your transaction:
+
+```typescript title="fee-payment-transaction.ts"
+--8<-- "code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction.ts:14:16"
+```
+
+### Create a Signer
+
+Create a signer using Alice's development account:
+
+```typescript title="fee-payment-transaction.ts"
+--8<-- "code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction.ts:18:29"
+```
+
+This function will return a signer that can be used to sign the transaction.
+
+### Setup the Client and API
+
+Create the client connection to the local Asset Hub:
+
+```typescript title="fee-payment-transaction.ts"
+--8<-- "code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction.ts:31:37"
+```
+
+### Create the Transaction
+
+Create a standard DOT transfer transaction:
+
+```typescript title="fee-payment-transaction.ts"
+--8<-- "code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction.ts:39:42"
+```
+
+This creates a transaction that transfers 3 DOT to Bob's address while keeping Alice's account alive.
+
+### Sign and Submit with Alternative Fee Payment
+
+The key part of this tutorial is specifying an alternative asset for fee payment. This is done through the `asset` parameter in the `signAndSubmit` options:
+
+```typescript title="fee-payment-transaction.ts"
+--8<-- "code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction.ts:44:69"
+```
+
+This specifies that the fees should be paid using the USDT asset.
+
+## Full Code
+
+The full code for the complete implementation is the following:
+
+??? code "Complete Code"
+
+    ```typescript title="fee-payment-transaction.ts"
+    --8<-- "code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction.ts"
+    ```
+
+## Running the Script
```

Comment:
Ideally this would be action-oriented "Run the Script" but I honestly don't think this needs to be a standalone section and could just be a step if this was swapped to an ordered list